lib/ansible/module_utils/rds.py

```diff
@@ -51,16 +51,18 @@ def __init__(self, dbinstance):
             self.name = None
         else:
             self.name = self.instance.get('DBInstanceIdentifier')
-        self.status = self.instance.get('DBInstanceStatus')
         self.data = self.get_data()
 
+    def status(self):
+        return self.instance.get('DBInstanceStatus')
```

Comment:
the call to self.instance.get was already a function.  Before this was an instance variable so the function would be called only once during object initialisation.  As it is now, whenever status is called self.instance.get('DBInstanceStatus') should be called as a function which I think _should_ help